### PR TITLE
The user who resets the password is now redirected to the login page.

### LINKED ISSUE #1899: User who resets a password see password reset form after resetting it
The user who resets the password should be directed to the login form after resetting the password.

screenshot from 2019-01-19 02-00-51

### PR DESCRIPTION
**Pull request by rajvaibhavdubey**
#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
The user who resets the password is now redirected to the login page.

Fixes #1899

### CODE REVIEW

File: app/components/forms/reset-password-form.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export default Component.extend(FormMixin, {
           this.get('loader')
             .patch('auth/reset-password', payload)
             .then(() => {
+              this.transitionToRoute('login');
```
Comment (CosmicCoder96) [inline app/components/forms/reset-password-form.js]: Place this line after line 58.

Comment (rajvaibhavdubey) [reply app/components/forms/reset-password-form.js]: Done :+1:
Thankyou :smile:

**Comment by rajvaibhavdubey** [comment]
@CosmicCoder96 How do I test the changes, because I'm not getting the password reset link. This should work I guess, just a simple redirect.

**Comment by CosmicCoder96** [comment]
@rajvaibhavdubey it doesn't and won't work. It's not a simple redirect. `this.transitionToRoute` In this context `this` refers to the component, component doesn't have a `transitionToRoute` method. 
https://emberjs.com/api/ember/3.7/classes/Component
Also, it wastes a lot of time of the reviewers when you send a PR without testing it for the bug it was supposed to solve in the first place. I don't understand why everyone is in a hurry to get code merged. Also, you don't need a verification link to test the single line of code this PR adds, put it inside any component, and it will throw an exception.

**Comment by ritikamotwani** [comment]
@rajvaibhavdubey any updates?

**Comment by sk9331657** [comment]
@CosmicCoder96  Claiming this due to inactivity.

**Pull request closed without merging**